rpc/ethereum/namespaces/eth/api.go

```diff
@@ -843,6 +843,14 @@ func (e *PublicAPI) GetTransactionReceipt(hash common.Hash) (map[string]interfac
 		receipt["contractAddress"] = crypto.CreateAddress(from, txData.GetNonce())
 	}
 
+	if dynamicTx, ok := txData.(*evmtypes.DynamicFeeTx); ok {
+		baseFee, err := e.backend.BaseFee(res.Height)
+		if err != nil {
+			return nil, err
```

Comment:
not sure if it is good to throw error here,
what if we want to disable basefee but still support eip 1559

Reply:
> not sure if it is good to throw error here, what if we want to disable basefee but still support eip 1559 transaction type

That's equivalent to enabling basefee, but has a constant base fee amount?

Reply:
nvm it seems that error is trigger only for internal error

Reply:
And it don't return error here if base fee is not enabled, it returns a zero and nil.